static/src/javascripts/projects/common/views/commercial/creatives/expandable-video-v2.html

```diff
@@ -0,0 +1,45 @@
+<div class="creative--expandable-video <%=data.videoOptions%>">
```

Comment:
This seems very dense... I don't know if we could perhaps mitigate this by splitting it into multiple partials or maybe at least just adding some whitespace and some comments annotating the different parts of the template.